### PR TITLE
Adding prometheus metrics description

### PR DESCRIPTION
**Pull request by amitkanfer**
Solving this issue:
https://github.com/open-policy-agent/opa/issues/2037

### CODE REVIEW

File: docs/content/monitoring.md [hunk 1]
```diff
@@ -26,6 +26,40 @@ scrape_configs:
       - "localhost:8181"
 ```
 
+Attaching the descrption for the prometheus metrics collected:
```
Comment (tsandall) [inline docs/content/monitoring.md:29]: Nit: I'd change this sentence to something like...

```
The Prometheus endpoint exports Go runtime metrics as well as HTTP request latency metrics for all handlers (e.g., `v1/data`).
```

**Review by tsandall — COMMENTED** [review]
Looks good! Just a small comment about the sentence above the table. Once that's fixed, we can merge this!

**Comment by tsandall** [comment]
@amitkanfer if you can amend your commit we'll merge this right away.

**Comment by amitkanfer** [comment]
@tsandall - ready for your re-review.

**Pull request merged**